proto/tendermint/abci/types.proto

```diff
@@ -435,7 +435,7 @@ message Validator {
 }
 
 message ValidatorUpdate {
-  tendermint.crypto.PublicKey pub_key = 1 [(gogoproto.nullable) = false];
+bytes pub_key = 1 ;
```

Comment:
Is it OK to lose the `gogoproto.nullable` attribute here?

Reply:
cant use nullable on a proto type

Reply:
Why not? Asking because we recently made use of it in another type.

Reply:
I think If you remove the `nullable` you will have to check for `nil` every time you access pub_key. Failure to do so might lead to a runtime panic due to a nil pointer dereference. Also tests would need to be adapted to handle this use case.